[[ Bug 22808 ]] Use default browser to show OAuth2 dialog on Linux

livecodepanos opened the pull request:
This patch uses the default browser instead of the browser widget to show OAuth2 dialog on Linux.

runrevmark commented on extensions/script-libraries/oauth2/oauth2.livecodescript:
```diff
@@ -1,4 +1,4 @@
-﻿script "com.livecode.library.oauth2"
+script "com.livecode.library.oauth2"
```
What's the reason for the change to the first line? (My guess is that a BOM has been added or removed)

livecodepanos replied:
Yeap, it seems it you edit a `.livecodescript` file in the IDE it removes the BOM.

I have fixed it now

runrevmark reviewed (changes requested):
PR title / commit message title should have prefix `[[ Bug 22808 ]]`

livecodepanos committed:
[[ Bug 22808 ]] Use default browser to show OAuth2 dialog on Linux

This patch uses the default browser instead of the browser widget
to show OAuth2 dialog on Linux.

FourthWorld commented:
I very much appreciate Panos jumping on this several months ago, but progress seems to have stalled out on the Vulcan test. Is there anything I might be able to help with to move this along?

runrevmark commented:
@livecode-vulcan review ok 47048e1

livecode-vulcan commented:
💙 review by @runrevmark ok 47048e11255d0d5ee1df2d7251e858785f8ac943

livecode-vulcan commented:
😎 test success 47048e11255d0d5ee1df2d7251e858785f8ac943

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/984)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/392)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/391)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/390)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/391)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2509)
* try-community-universal-ios-iphoneos14.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos14.4/builds/38)
* try-community-universal-ios-iphoneos14.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos14.1/builds/87)
* try-community-universal-ios-iphonesimulator14.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator14.4/builds/38)
* try-community-universal-ios-iphonesimulator14.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator14.1/builds/85)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2536)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2525)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2523)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2560)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2496)
